docs/INSTALL_HUB_SPOKE.md

```diff
@@ -72,32 +68,48 @@ services:
       - DOCKER_INFLUXDB_INIT_PASSWORD=${PASSWORD}
       - DOCKER_INFLUXDB_INIT_ORG=homelab
       - DOCKER_INFLUXDB_INIT_BUCKET=scrutiny
-      - DOCKER_INFLUXDB_INIT_ADMIN_TOKEN=your-very-secret-token
+      - DOCKER_INFLUXDB_INIT_ADMIN_TOKEN=SUPER-SECRET-TOKEN
+      - TZ=Europe/Stockholm
     restart: unless-stopped
     networks:
       - monitoring
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8086/health"]
+      interval: 5s
+      timeout: 10s
+      retries: 20
```

Comment:
is this necessary? I think there's a healthcheck built into the image.

Reply:
Not sure actually, but let's remove it. :)